README.md

```diff
@@ -14,6 +14,11 @@ PlanetScale is more than a database and our CLI is more than a jumble of command
 ```
 brew install planetscale/tap/pscale
 ```
+!! Note of importance: `pscale` requires the MySQL Client. You can install it by running:
```

Comment:
We only require it for the `pscale shell` subcommand, otherwise, it's optional. I wonder if should add this bit in paranthese? 

```suggestion
Optional: `pscale` requires the MySQL Client for certain commands. You can install it by running:
```

Reply:
As a user, it would make perfect sense to me so I'm keeping my eyes peeled to see what you decide.

Reply:
Thank you Edgar.

Reply:
Your welcome Fatih, its an awesome db platform😄